src/command/verify/verify.c

```diff
@@ -132,8 +154,80 @@ typedef struct VerifyJobData
     bool enableArchiveFilter;                                       // Only check archives in the specified range
     const String *archiveStart;                                     // Start of the WAL range to be verified
     const String *archiveStop;                                      // End of the WAL range to be verified
+    VariantList *errorList;                                                // List of errors that occurred during the job
 } VerifyJobData;
 
+/***********************************************************************************************************************************
+Helper function to add a string with log level to an error list
+***********************************************************************************************************************************/
+
+static void
+verifyErrorNew(VariantList *errorList, const LogLevel logLevel, const String *const message)
+{
+    FUNCTION_TEST_BEGIN();
+        FUNCTION_TEST_PARAM(VARIANT_LIST, errorList);
+        FUNCTION_LOG_PARAM(ENUM, logLevel);                        // Log level
```

Comment:
```suggestion
        FUNCTION_LOG_PARAM(ENUM, logLevel);                         // Log level
```
Fix indents of comments here and in other places. To make it equal with other code